### PR TITLE
Make Lighthouse/YLT/Pa11y work in GH Action Docker regression test

### PR DESCRIPTION
**Pull request by krompaco**
With these fixes Test 1 now seems to run fine in GH actions, see job log here:
https://github.com/krompaco/webperf_core/actions/runs/11741517843/job/32710317461

I think that this fix also gives Test 17+18 a chance to work. Just need to figure out how to get the --no-sandbox flag set when launching Chrome from Pa11y and YLT.

I had issues with the repo on Windows after syncing fork.
Fix was to delete the old casing version of CREDITS.md.

### CODE REVIEW

**Comment by 7h3Rabbit** [comment]
@krompaco Is this our solution?
https://github.com/pa11y/pa11y-ci?tab=readme-ov-file#docker
We already have pa11y-config.json (but doesn't seem to use it).

File: .github/workflows/regression-test-docker-image.yml [hunk 1]
```diff
@@ -74,16 +74,16 @@ jobs:
     - name: Check if webperf_core can run Test 1
       if: always()
       run: |
-        testresult=$(docker run ${{ env.TEST_TAG }} python default.py -t 1 -r -i sites.json)
+        testresult=$(docker run --shm-size=4g -e MAX_OLD_SPACE_SIZE=3000 ${{ env.TEST_TAG }} python default.py -t 1 -r -i sites.json)
```
Comment (7h3Rabbit) [inline .github/workflows/regression-test-docker-image.yml:77]: @krompaco cool, maybe we should set it using ENV in docker file instead?
What is the lowest value that works?

Comment (7h3Rabbit) [reply .github/workflows/regression-test-docker-image.yml:77]: @krompaco IF arguments you use here is required (I would prefer a way it was not required), this PR should also include modification of docker documentation to include this.

File: .github/workflows/regression-test-docker-image.yml [hunk 2]
```diff
@@ -220,7 +220,7 @@ jobs:
     - name: Check if webperf_core can run Test 18
       if: always()
       run: |
-        testresult=$(docker run ${{ env.TEST_TAG }} python default.py -t 18 -r -i sites.json)
+        testresult=$(docker run --shm-size=4g -e MAX_OLD_SPACE_SIZE=3000 ${{ env.TEST_TAG }} python default.py -t 18 -r -i sites.json)
```
Comment (7h3Rabbit) [inline .github/workflows/regression-test-docker-image.yml:223]: @krompaco Test 22 also need this change (as it use lighthouse performance in the background)

File: .github/workflows/regression-test-docker-image.yml (hunk 1 above)
Comment (krompaco) [reply .github/workflows/regression-test-docker-image.yml:77]: They shouldn't be required on "larger" host machines but seems to be needed for GH Actions where heavier work is done. Values are dependent on host machine resources, so not a good candidate to "burn" into the image.

**Comment by krompaco** [comment]
> @krompaco Is this our solution? https://github.com/pa11y/pa11y-ci?tab=readme-ov-file#docker We already have pa11y-config.json (but doesn't seem to use it).

I think I fixed YLT and Pa11y now too. If file is called ./pa11y.json it gets picked up. I also found an ENV-variable that YLT => phantomas detects and sets --no-sandbox. I used same parameters in pa11y.json as that trick results in.
https://github.com/krompaco/webperf_core/actions/runs/11755587585/job/32750791723

ping @7h3Rabbit

File: .github/workflows/regression-test-docker-image.yml (hunk 2 above)
Comment (krompaco) [reply .github/workflows/regression-test-docker-image.yml:223]: Ah, thanks. Should be added now.

**Comment by 7h3Rabbit** [comment]
@krompaco Only modification of docker documentation to include params required before approval and merge :)

File: pa11y-docker-config.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (7h3Rabbit) [inline pa11y-docker-config.json:1]: @krompaco Is it possible to move this into docker folder?

**Pull request merged**